frontend/mr-admin-flate/src/components/tiltaksgjennomforinger/TiltaksgjennomforingSkjemaPage.tsx

```diff
@@ -0,0 +1,139 @@
+import { useUtkast } from "../../api/utkast/useUtkast";
+import { TiltaksgjennomforingSkjemaContainer } from "./TiltaksgjennomforingSkjemaContainer";
+import React from "react";
+import { useTiltaksgjennomforing } from "../../api/tiltaksgjennomforing/useTiltaksgjennomforing";
+import styles from "../skjema/Skjema.module.scss";
+import { Alert } from "@navikt/ds-react";
+import { ErrorMeldinger } from "./TiltaksgjennomforingSkjemaErrors";
+
+const TiltaksgjennomforingSkjemaPage = () => {
+  const navigate = useNavigate();
+  const [searchParams] = useSearchParams();
+  const queryClient = useQueryClient();
+  // const [error, setError] = useState<React.ReactNode | null>(null);
+  const error = () => {
+    return (
+      !avtale ||
+      (avtale &&
+        avtale?.sluttDato &&
+        new Date(avtale.sluttDato) < new Date()) ||
+      (avtale && !avtale?.navRegion) ||
+      isErrorAnsatt ||
+      isErrorEnheter
+    );
+  };
+
+  const { data: avtale } = useAvtale(
+    searchParams.get("tiltaksgjennomforingId") || undefined,
+  );
+
+  const {
+    data: tiltaksgjennomforing,
+    isFetching: tiltaksgjennomforingFetching,
+  } = useTiltaksgjennomforing(
+    searchParams.get("tiltaksgjennomforingId") || undefined,
+  );
+  const { data: utkast, isFetching: utkastFetching } = useUtkast(
+    searchParams.get("utkastId") || undefined,
+  );
+  const { data: tiltakstyper, isLoading: isLoadingTiltakstyper } =
+    useTiltakstyper({ status: Tiltakstypestatus.AKTIV }, 1);
+  const {
+    data: ansatt,
+    isLoading: isLoadingAnsatt,
+    isError: isErrorAnsatt,
+  } = useHentAnsatt();
+  const {
+    data: enheter,
+    isLoading: isLoadingEnheter,
+    isError: isErrorEnheter,
+  } = useAlleEnheter();
+
+  const utkastModus = utkast && inneholderUrl(utkast?.id);
+  const redigeringsModus =
+    utkastModus ||
+    (tiltaksgjennomforing && inneholderUrl(tiltaksgjennomforing?.id));
+
+  const navigerTilbake = () => {
+    navigate(-1);
+  };
+
+  if (utkastFetching || tiltaksgjennomforingFetching) {
+    return (
+      <Laster
+        size="xlarge"
+        tekst={
+          utkastFetching ? "Laster utkast..." : "Laster tiltaksgjennomføring..."
+        }
+      />
+    );
+  }
+
+  return (
+    <>
+      <main>
+        <Header
+          dataTestId={
+            redigeringsModus
+              ? "rediger-tiltaksgjennomforing-header"
+              : "opprett-tiltaksgjennomforing-header"
+          }
+        >
+          {redigeringsModus
+            ? "Rediger tiltaksgjennomforing"
+            : "Opprett ny tiltaksgjennomforing"}
+        </Header>
+        <ContainerLayoutDetaljer>
+          <div className={styles.skjema}>
+            {isLoadingAnsatt || isLoadingTiltakstyper || isLoadingEnheter ? (
+              <Laster />
+            ) : null}
+            <div className={styles.skjema_content}>
+              {error() &&
+              (redigeringsModus !== undefined || redigeringsModus) ? (
```

Comment:
Holder det ikke å skrive `(redigeringsmodius ? .....`? Må du sjekke for undefined også?